index.html

```diff
@@ -101,9 +148,6 @@ <h3>Meet the team:</h3>
             </ul>
         </div>
     </footer>
-
-    <!-- Custom JS Files -->
-    <script src="assets/js/main.js" defer></script>
```

Comment:
The only thing I would argue here is the moving of the JS file script tag to the top of the HTML? Happy to be proven wrong here, however with the evolution of JS in recent years, maybe I'm just out of date? 😃 🦖

Reply:
The defer tag causes the script to be downloaded in parallel with the rest of the page (doesn't block) and wait until the from is ready for execution. So placing a defer script in the head should download and be ready as fast or faster than planning it at the end.